README.adoc

```diff
@@ -176,6 +176,17 @@ No further intervention is required in the success case.
 
 WARNING: `rpi-sb-provisioner` will not, by default, block JTAG access. If you wish to make use of this facility, you _must_ specify this in the Raspberry Pi Bootloader configuration pointed to by `RPI_DEVICE_BOOTLOADER_CONFIG_FILE`
 
+=== Monitoring via the app
+
+`rpi-sb-provisioner` also contains a monitoring TUI. This can be used to observe how far through the process each device is. It also allows for easy introspection of the log files and lists all completed and failed devices.
+
+To run, type into a terminal window: `monitor.sh`
```

Comment:
The preferred style is to have the command on a separate line with the user indicator, eg:

----
$ monitor.sh
----